### PR TITLE
worked on login component

### PR DESCRIPTION
**Pull request by ValentinCop**

### CODE REVIEW

**Review by ghost — DISMISSED** [review]
Good for me

File: frontend/src/components/Login/index.jsx [hunk 1]
```diff
@@ -1,13 +1,50 @@
...
+  const hChange = (evt) => {
+    const { name, value } = evt.target;
+    setFormData({ ...formData, [name]: value });
+  };
+
+  const hSubmit = (evt) => {
+    evt.preventDefault();
+    axios
+      .post(`${import.meta.env.VITE_BACKEND_URL}/auth/login`, formData)
+      .then(() => {})
```
Comment (LuigiMenez) [inline frontend/src/components/Login/index.jsx]: Error handling, would be nice

**Review by LuigiMenez — CHANGES_REQUESTED** [review]
Error handling, would be nice

**Review by ghost — DISMISSED** [review]
Ok for me

**Review by LuigiMenez — DISMISSED** [review]
Ok for me.

File: frontend/src/components/Login/index.jsx [hunk 2]
```diff
@@ -1,13 +1,54 @@
...
+    setFormData({ ...formData, [name]: value });
+  };
+
+  const hSubmit = (evt) => {
+    evt.preventDefault();
+    axios
+      .post(`${import.meta.env.VITE_BACKEND_URL}/auth/login`, formData)
+      .then(() => {
+        console.warn("welcome");
+      })
```
Comment (LoicBrassart) [inline frontend/src/components/Login/index.jsx]: ... Just nope:
- Useless
- Misuse of `console.warn` (you use it as a `console.log` authorized by ESLint :cry: )

File: frontend/src/components/Login/index.jsx [hunk 3]
```diff
@@ -1,13 +1,54 @@
...
+      .catch(() => {
+        console.error("wrong login");
```
Comment (LoicBrassart) [inline frontend/src/components/Login/index.jsx]: Even this is not really useful, you could instead dispatch a Toast notification ?

**Review by LoicBrassart — CHANGES_REQUESTED** [review]
Fix this

**Review by ghost — DISMISSED** [review]
ok for me

**Review by ghost — DISMISSED** [review]
ok for me

**Review by LuigiMenez — DISMISSED** [review]
Ok for me.

**Comment by LoicBrassart** [comment]
Missing validations

**Review by ghost — APPROVED** [review]
Ok for me

**Pull request closed without merging**